src/general-functions/renderModalElements.js

```diff
@@ -0,0 +1,5 @@
+import ModalElements from "../components/modal-components/modal-elements";
+
+const renderModalElements = (labels) => labels.map(label => <ModalElements label = {label} />) 
```

Comment:
it's a good practice to add key when rendering multiple components with map method